configure.ac

```diff
@@ -2350,6 +2350,7 @@ AC_SUBST(BUILD_WITH_LIBDBI_LIBS)
 # --with-libdpdk {{{
 AC_ARG_VAR([LIBDPDK_CPPFLAGS], [Preprocessor flags for libdpdk])
 AC_ARG_VAR([LIBDPDK_LDFLAGS], [Linker flags for libdpdk])
+AC_ARG_VAR([LIBDPDK_LIBS], [Library to link for libdpdk])
```

Comment:
Surely - and in general thanks for your review.
I can make build systems work, but lack the in depth experience in some of them - so thanks for pointing out all the improvement chances - addressing them one by one now.